install/website-chart.js.2.5.0.patch

```diff
@@ -0,0 +1,51 @@
+--- Chart.js	2017-05-25 09:56:43.000000000 +0200
++++ website/js/Chart.js	2017-10-18 09:22:45.000000000 +0200
+@@ -8868,6 +8868,7 @@
+ 			beforeLabel: helpers.noop,
+ 			label: function(tooltipItem, data) {
+ 				var datasetLabel = data.datasets[tooltipItem.datasetIndex].label || '';
++				var customLabel = data.datasets[tooltipItem.datasetIndex].data[tooltipItem.index].label || ''; 
```

Comment:
rather than patching this library, i think you can just override the `label` function from outside.. a bit hacky but slightly less hacky than the patching